pkg/osquery/interactive/interactive.go

```diff
@@ -0,0 +1,98 @@
+package interactive
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/kit/fs"
+	"github.com/kolide/launcher/pkg/osquery/table"
+	osquery "github.com/osquery/osquery-go"
+)
+
+const extensionName = "com.kolide.launcher_interactive"
+
+func StartProcess(rootDir, osquerydPath string, osqueryFlags []string) (*os.Process, error) {
+
+	if err := os.MkdirAll(rootDir, fs.DirMode); err != nil {
+		return nil, fmt.Errorf("creating root dir for interactive mode: %w", err)
+	}
+
+	// Get the current working directory.
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil, fmt.Errorf("error getting current working directory: %w", err)
+	}
+
+	// Transfer stdin, stdout, and stderr to the new process
+	// and also set target directory for the shell to start in.
+	pa := os.ProcAttr{
+		Files: []*os.File{os.Stdin, os.Stdout, os.Stderr},
+		Dir:   cwd,
```

Comment:
I'll take it, but it seems weird. Why do we need that? Shouldn't it be the default if we're not changing directories?

Reply:
you're correct, no need for that ... removed